src/review/views.py

```diff
@@ -799,6 +825,66 @@ def do_review(request, assignment_id):
     return render(request, template, context)
 
 
+@require_POST
+@reviewer_user_for_assignment_required
+def upload_review_file(request, assignment_id):
+    access_code = logic.get_access_code(request)
+
+    if access_code:
+        assignment = models.ReviewAssignment.objects.get(
+            Q(pk=assignment_id) &
```

Comment:
PEP 8 recommends operators to be leading the line rather than trailing for improved readability:

https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
Done.